MOBT-5-Dialog-Messenger/MOBT-5-Dialog-Messenger/Chat/ChatLogMessageCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ChatLogMessageCell.swift
+//  MOBT-5-Dialog-Messenger
+//
+//  Created by Mykyta Babanin on 20.04.2022.
+//
+
+import UIKit
+
+class ChatLogMessageCell: BaseCell {
+    
+    lazy var messageTextView = UITextView()
```

Comment:
must be private

Reply:
messageTextView was used in chatLogController when I dequeued cell.